sprokit/processes/adapters/CMakeLists.txt

```diff
@@ -42,18 +13,23 @@ set( lib_sources
   embedded_pipeline.cxx
 
   adapter_base.h             adapter_base.cxx
-  input_adapter_process.h    input_adapter_process.cxx
-  output_adapter_process.h   output_adapter_process.cxx
+  input_adapter_process.cxx
+  output_adapter_process.cxx
   )
 
 set( public_headers
   adapter_types.h
   adapter_data_set.h
   input_adapter.h
+  input_adapter_process.h
```

Comment:
I'm going to merge this for testing on the dashboard tonight, then I can revisit making these headers private.